src/js/common/schemaform/AskVAQuestions.jsx

```diff
@@ -0,0 +1,32 @@
+import PropTypes from 'prop-types';
```

Comment:
Maybe a bit premature, but this doesn't look like it's schemaform-specific (though it is only being used in schemaform for now, I realize). Might it be worth it to put this in `src/js/common/components`?